src/connector/src/sink/remote.rs

```diff
@@ -64,28 +67,54 @@ use crate::ConnectorParams;
 macro_rules! def_remote_sink {
     () => {
         def_remote_sink! {
-            { ElasticSearch, ElasticSearchSink, "elasticsearch" },
-            { Cassandra, CassandraSink, "cassandra" },
-            { Jdbc, JdbcSink, "jdbc" },
+            { ElasticSearch, ElasticSearchSink, "elasticsearch" }
+            { Cassandra, CassandraSink, "cassandra" }
+            { Jdbc, JdbcSink, "jdbc", |desc| {
+                desc.sink_type.is_append_only()
```

Comment:
Are we planning to enable upsert jdbc sink decoupling as well?

Reply:
May keep the same as other sinks like kafka, kinesis for now. Current implementation does not support rewind on failure yet, and enabling it by default does not help resolve much of the current problems. 

I think we can enable sink decouple for these sinks by default after rewind on failure is supported.